aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseMVC/Models/DateRange.cs

```diff
@@ -12,7 +12,8 @@ public DateRange(DateOnly fromDate, DateOnly toDate)
             To = toDate;
         }
 
-        public static bool TryParse(string? value, IFormatProvider? provider, out DateRange? dateRange)
+        public static bool TryParse(string? value, IFormatProvider? provider,
```

Comment:
![image](https://user-images.githubusercontent.com/3605364/179855688-921214aa-97f6-497f-8c9f-6d4e153d3ef1.png)
@fiyazbinhasan 
 Wrong provider

Reply:
Error message is great
<img width="734" alt="image" src="https://user-images.githubusercontent.com/3605364/179855834-d0bd97cb-998d-4c19-a88d-38aeac5a535e.png">